src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/VSResources.resx

```diff
@@ -208,4 +208,36 @@ Press Ignore to ignore the external changes. Your changes may be lost if you clo
   <data name="AnalyzersNodeName" xml:space="preserve">
```

Comment:
Why the rename to VSResources? This seems to be generating a lot of diff noise, what does this get us?